### PR TITLE
Parameterize candidate_name and count for funding_sources SQL request

### PR DESCRIPTION
**Pull request by RaquelFigueroa**
**1. Prevent SQL injection in raw SQL funding_sources query by parameterizing request.**

**2. Link to Trello Ticket: None at the moment**

**3. This parameterization will need to be applied to all SQL request models to guard against SQL injection vulnerabilities.**

**4. Remember to tag reviewers!**

### CODE REVIEW

File: django/congressionaldata/api/views.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def funding_sources(request, candidate_name, count):
         WHERE recipient_candidate_name LIKE %s
         GROUP BY donor_name
         ORDER BY sum DESC LIMIT %s
-        """, ["%%%s%%" % candidate_name.upper(), str(count)])
+        """, params = (["%%%s%%" % candidate_name.upper(), str(count)]))
```
Comment (erikeldridge) [inline django/congressionaldata/api/views.py]: This change names the param argument, but per the [django docs](https://docs.djangoproject.com/en/2.0/topics/db/sql/#passing-parameters-into-raw) I think we were already parameterizing the inputs.

I think the larger issue is we're making raw sql queries, which requires us to be sophisticated enough to know we should be parameterizing inputs, hence the warnings on the raw sql doc ("You should be very careful whenever you write raw SQL").

Django provides a rich model layer that performs these best-practices implicitly, so the larger task is to migrate us away from using raw sql for common queries: https://docs.djangoproject.com/en/2.0/topics/db/queries/#retrieving-objects.

**Comment by erikeldridge** [comment]
I added @swetagsanghavi to this review since she's spinning up on the web app side. So you're not blocked on any one person for review, plz add Sweta and @EdwardVillasenor1992 to django-related changes.

File: django/congressionaldata/api/views.py (hunk 1 above)
Comment (EdwardVillasenor1992) [inline django/congressionaldata/api/views.py]: no changes needed

Comment (EdwardVillasenor1992) [inline django/congressionaldata/api/views.py]: no changes needed

**Pull request merged**